src/useMemoryRouter.tsx

```diff
@@ -34,23 +36,29 @@ export const useMemoryRouter = (singletonRouter: MemoryRouter, eventHandlers?: M
   // Subscribe to any eventHandlers:
   useEffect(() => {
     if (!eventHandlers) return;
-    const { onRouteChangeStart, onRouteChangeComplete, onPush, onReplace } = eventHandlers;
-    if (!(onRouteChangeStart || onRouteChangeComplete || onPush || onReplace)) return;
+    const { onRouteChangeStart, onRouteChangeComplete, onHashChangeComplete, onHashChangeStart, onPush, onReplace } = eventHandlers;
+    if (!(onRouteChangeStart || onRouteChangeComplete  || onHashChangeStart ||onHashChangeComplete || onPush || onReplace)) return;
```

Comment:
Minor nit:
```suggestion
    if (!(onRouteChangeStart || onRouteChangeComplete  || onHashChangeStart || onHashChangeComplete || onPush || onReplace)) return;
```
I should add precommit hooks or something, this should be automated

Reply:
I removed this if statement and ran prettier